readium/shared/src/main/java/org/readium/r2/shared/fetcher/Resource.kt

```diff
@@ -24,6 +24,7 @@ import org.readium.r2.shared.extensions.requireLengthFitInt
 import org.readium.r2.shared.parser.xml.ElementNode
 import org.readium.r2.shared.parser.xml.XmlParser
 import org.readium.r2.shared.publication.Link
+import org.readium.r2.shared.publication.services.cover
```

Comment:
What the fuck is that? There are a few unused imports through the code base, that one is the worst one I could see.

Reply:
Yeah a right-click of the readium directory and Optimize Imports should always be done.  Wonder if there is a Gradle task or checker.

Reply:
I'm not sure where it came from either, [the commit that introduced it doesn't use it](https://github.com/readium/kotlin-toolkit/commit/07d58d32cc969c9e4704331e96a57a4871c3f249#diff-9f256285c5d2609af688ffd63eff2b7990c47301bc7422a20a823fd87354166b). Maybe Android Studio auto-importing something with a failed auto-completion?

I didn't know about this global optimize imports, that's helpful thanks @stevenzeck. I'll do this for this PR but it's about time we setup an auto-formatter like `ktlint` which tackles that automatically.